### PR TITLE
add peers aura and acre

### PR DESCRIPTION
**Pull request by konsortech**

### CODE REVIEW

File: acrechain/chain.json [hunk 1]
```diff
@@ -213,7 +225,7 @@
       {
         "address": "https://acrechain-grpc.ramuchi.tech:1090",
         "provider": "ramuchi.tech"
-      }
+      },
```
Comment (JeremyParish69) [inline acrechain/chain.json:228]: remove comma

File: aura/chain.json [hunk 2]
```diff
@@ -158,7 +171,7 @@
       {
         "address": "aura.grpc.kjnodes.com:17090",
         "provider": "kjnodes"
-      }
+      },
```
Comment (JeremyParish69) [inline aura/chain.json:174]: remove comma

File: acrechain/chain.json [hunk 3]
```diff
@@ -121,7 +121,11 @@
         "id": "82726047c95e4ddf9a0fa82e4c4c4f17cbf3f140",
         "address": "213.32.24.201:27656",
         "provider": "genznodes"
-      }
+      },
+      {
+        "id": "7d630b6e517598b4dc84a07c15fe328709a2705b",
+        "address": "mainnet-acre.konsortech.xyz:13656",
+        "provider": "KonsorTech",
```
[the remark this answers is not in the document]
Comment (konsortech) [reply acrechain/chain.json]: @JeremyParish69 fixed

File: acrechain/chain.json (hunk 1 above)
Comment (konsortech) [reply acrechain/chain.json]: @JeremyParish69 fixed

**Pull request merged**